src/components/menu/examples/menu-basic-secondary-text.tsx

```diff
@@ -0,0 +1,66 @@
+import { MenuItem, ListSeparator } from '@limetech/lime-elements';
+import { Component, h, State } from '@stencil/core';
+
+/**
+ * With `secondaryText`
+ *
+ * Menu items can display secondary text as well. By default, the secondary text
+ * will be displayed in two lines, and then get truncated.
+ *
+ * :::important
+ * Keep in mind that a menu's drop-down will stretch as much as it can to display
+ * their content. You must therefore specify a maximum width for menus with
```

Comment:
```suggestion
 * its content. You must therefore specify a maximum width for menus with
```

Asch, I missed this one. But I'll just commit it as a fixup myself!